package-testing/python-sdk-relay/src/server.py

```diff
@@ -0,0 +1,169 @@
+        "sdkName": "python-sdk", 
+        "sdkVersion": "4.1.0",
+        "supportsBandits": False,
+        "supportsDynamicTyping": False
+    })
+
+@app.route('/flags/v1/assignment', methods=['POST'])
+def handle_assignment():
+    data = request.json
+    request_obj = AssignmentRequest(
+        flag=data['flag'],
+        subject_key=data['subjectKey'],
+        subject_attributes=data['subjectAttributes'],
+        assignment_type=data['assignmentType'],
+        default_value=data['defaultValue']
+    )
+    print(f"Request object: {request_obj}")
+    
+    client = eppo_client.get_instance()
+    
+    try:
+        match request_obj.assignment_type:
+            case 'BOOLEAN':
+                result = client.get_boolean_assignment(
+                    request_obj.flag, 
+                    request_obj.subject_key, 
+                    request_obj.subject_attributes, 
+                    bool(request_obj.default_value)
+                )
+            case 'INTEGER':
+                result = client.get_integer_assignment(
+                    request_obj.flag, 
+                    request_obj.subject_key, 
+                    request_obj.subject_attributes, 
+                    int(request_obj.default_value)
+                )
+            case 'STRING':
+                result = client.get_string_assignment(
+                    request_obj.flag, 
+                    request_obj.subject_key, 
+                    request_obj.subject_attributes, 
+                    request_obj.default_value
+                )
+            case 'NUMERIC':
+                result = client.get_numeric_assignment(
+                    request_obj.flag, 
+                    request_obj.subject_key, 
+                    request_obj.subject_attributes, 
+                    float(request_obj.default_value)
+                )
+            case 'JSON':
+                result = client.get_json_assignment(
+                    request_obj.flag, 
+                    request_obj.subject_key, 
+                    request_obj.subject_attributes, 
+                    request_obj.default_value
+                )
+        
+        response = {
+            "result": result,
+            "assignmentLog": [],
+            "banditLog": [],
+            "error": None
+        }
+        print(f"response: {response}")
+        return jsonify(response)
+    except Exception as e:
+        print(f"Error processing assignment: {str(e)}")
+        response = {
+            "result": None,
+            "assignmentLog": [],
+            "banditLog": [],
+            "error": str(e)
+        }
+        return jsonify(response)
+
+@dataclass
+class BanditActionRequest:
+    flag: str
+    subject_key: str
+    subject_attributes: dict
+    actions: list
+    default_value: any
+
+
+@app.route('/bandits/v1/action', methods=['POST'])
+def handle_bandit():
+    data = request.json
+    request_obj = BanditActionRequest(
+        flag=data['flag'],
```

Comment:
nitpicking, if runner won't send the data in payload perfectly you will run into 500. A nice little `validate_payload()` function would be 100/100 pts :))